pages/bare_metal_cloud/dedicated_servers/partitioning_ovh/guide.en-gb.md

```diff
@@ -210,30 +401,116 @@ In order to make the OVHcloud API as simple as possible, customers must define a
 |RAID 7|raidz3|4|3|7|
 |RAID 10|❌|4|0|4|
 
-### Partitioning Backend
+#### API & Partitioning
+
+Example of OS installation with a custom partitioning layout:
+
+> [!api]
+>
+> @api {v1} /dedicated/server POST /dedicated/server/{serviceName}/reinstall
+>
+
+```json
+{
+    "operatingSystem": "debian12_64",
+    "storage": [
+        {
+            "partitioning": {
+                "disks": 2,
+                "layout": [
+                    {
+                        "fileSystem": "ext4",
+                        "mountPoint": "/boot",
+                        "size": 1000,
+                        "raidLevel": 0
+                    },
+                    {
+                        "fileSystem": "ext4",
+                        "mountPoint": "/",
+                        "size": 0
+                    }
+                ]
+            }
+        }
+    ]
+}
+```
+
+`size` 0 means that the partition will fill all the remaining space on the involved disks.
 
-#### Architecture
+> [!primary]
+>
+> Up to 1 partition can be configured to fill the disk (size 0).
+>
 
-There is an abstraction layer between the partitioning description on the OVHcloud API (that is quite minimalist) and the real partitioning (that can be very complex).
+<br />
 
-The key feature of this partitioning is the separation between the **pre-processing** and the **action script**:
+> [!primary]
+>
+> If not specified `raidLevel` will be set to 1.
+>
 
-![Diagram Components](images/Diagram-Components-Simplified.svg){.thumbnail}
+In this example, only the 2 first disks of the disk group will be involved in the partitioning (ie. in the software RAID).
 
-- the **pre-processing** runs on the OVHcloud infrastructure. It just converts the data from the OVHcloud API to a richer JSON representation.
+> [!primary]
+>
+> `disks` parameter can be omitted if you want to involved all the disks of the disk group in the software RAID.
+>
 
-- the **action script** runs on the customer's server, takes the rich JSON representation of the partitioning and creates the partitions.
+As you can see, a partitioning layout is a list of partitions. Here is an example of a partition structure:
+
+```json
+{
+    "mountPoint": "/var/lib/vz",
+    "fileSystem": "ext4",
+    "size": 0,
+    "raidLevel": "1",
+    "extras": {
+        "lv": {
+            "name": "data"
+        }
+    }
+}
+```
+
+`extras` sub-hash is optional. For the moment it can only be used to specify that the partition will be a logical volume and its logical volume. It can also be used for ZFS:
```

Comment:
```suggestion
The `extras` sub-hash is optional. For the moment it can only be used to specify that the partition will be a logical volume and its logical volume. It can also be used for ZFS:
```
"a logical volume and its logical volume" : is this intended?